src/convert.cpp

```diff
@@ -38,7 +38,14 @@ ddwaf_object* to_ddwaf_object_array(ddwaf_object *object, Napi::Env env, Napi::A
   return object;
 }
 
-ddwaf_object* to_ddwaf_object_object(ddwaf_object *object, Napi::Env env, Napi::Object obj, int depth, bool lim) {
+ddwaf_object* to_ddwaf_object_object(
```

Comment:
shouldn't we make the same changes to the `to_ddwaf_object_array` function just above ?